packages/react-components/src/components/Attachment/AttachmentCardGroup.tsx

```diff
@@ -42,14 +44,15 @@ export interface _AttachmentCardGroupProps {
  * Renders the children equally spaced in multiple rows.
  */
 export const _AttachmentCardGroup = (props: _AttachmentCardGroupProps): JSX.Element => {
-  const { children, ariaLabel, attachmentGroupLayout } = props;
+  const { children, ariaLabel, attachmentGroupLayout, disabled } = props;
   if (!children) {
     return <></>;
   }
   return (
     <Stack
       horizontal
       className={mergeStyles(
+        disabled && attachmentGroupDisabled,
```

Comment:
not sure where i should ask this, but how does a11y work when card is disabled and how will it be announced when selected?

Reply:
good point, i missed it, let me add that in

Reply:
this would actually require public API change for new string, will include it in second PR instead